msteams-platform/toolkit/add-capability.md

```diff
@@ -66,12 +66,13 @@ Apart from the capabilities your Teams app already have, you can choose to add d
 |Existing capabilities|Other supported capabilities can be added|
 |--------------------|--------------------|
 |Tabs with SPFx|None|
-|Tabs with Azure|Bots and messaging extensions|
-|Bots|Tabs|
-|Messaging extensions|Tabs|
-|Tabs and bots|None|
-|Tabs and messaging extensions|None|
-|Tabs, bots, and messaging extensions|None|
+|Tabs with Azure|Bot and messaging extension|
+|Bot|Tabs|
+|Messaging extensions|Tabs and bot|
```

Comment:
maybe still use a singular form for `messaging extension`

Reply:
Ok, had updated the 'Other supported capabilities can be added'. I will change the Existing capabilities as well.